pyEpiabm/pyEpiabm/sweep/spatial_sweep.py

```diff
@@ -264,3 +251,39 @@ def do_infection_event(self, infector: Person, infectee: Person,
         r = random.random()
         if r < force_of_infection:
             infectee.microcell.cell.enqueue_person(infectee)
+
+    def near_neighbour(self, cell, other_cells):
+        '''
+        Helper function which takes in a given cell and the list of all cells
+        and generates a list of near neighbours which are the cells that are
+        closer than the cutoff for cross-cell infection
+
+        Parameters
+        ----------
+        cell : Cell
+            Cell to check for near neighbours
+        other_cells : typing.List[Cell]
+            List of all cells except cell
+
+        Returns
+        -------
+        cell.nearest_neighbours
+            Dictionary of all cells with distance below cutoff
+            Dictionary stores cell.id and distance between the 2 cells
+            These are stored in the form:
+            cell.id: distance
+        '''
+        # List of near neghbours, cells which are closer than the
+        # cutoff for cross-cell infection
+        cutoff = Parameters.instance().infection_radius
+
+        for cell2 in other_cells:
+            distance = DistanceFunctions.dist(cell.location, cell2.location)
```

Comment:
Here you are calculating the distance between two cells, but in L124 and also previous codes, it is the distance weight, which is one over the distance (ZeroDivisionError may occur).

Reply:
That's okay, but make sure you update the docs to reflect this change! Also be aware that the inverse is used to weight our choice of cells within the cutoff (so shorter distances have a higher probability) so you should make sure you still do this

Reply:
Where does this happen?

Reply:
We have changed this back now so docs don't need updating